pkg/webhook/admission/pod/mutator.go

```diff
@@ -54,6 +54,10 @@ func (mutator *Mutator) Handle(ctx context.Context, req types.Request) types.Res
 		return admission.ErrorResponse(http.StatusInternalServerError, err)
 	}
 
+	// For some reason pod namespace is always empty when coming to this pod mutator, need to get from admission request
+	if pod.Namespace == "" {
+		pod.Namespace = req.AdmissionRequest.Namespace
```

Comment:
Nice. Do we even need to check pod.Namespace == "" or can we just do the assignment?